packages/contracts/test/multi_sig_with_time_lock.ts

```diff
@@ -169,7 +183,8 @@ describe('MultiSigWalletWithTimeLock', () => {
                 txHash = await multiSig.confirmTransaction.sendTransactionAsync(txId, {
                     from: owners[1],
                 });
-                const confRes = await zeroEx.awaitTransactionMinedAsync(txHash);
+                await web3Wrapper.awaitTransactionMinedAsync(txHash, constants.AWAIT_TRANSACTION_MINED_MS);
+                const confRes = await zeroEx.awaitTransactionMinedAsync(txHash, constants.AWAIT_TRANSACTION_MINED_MS);
```

Comment:
`confRes` is not a very descriptive variable name. What about using the types name `transactionReceiptWithDecodedLogs` or `transactionReceipt` here instead? I know verbosity of variable names is a stylistic thing, but it does have a real impact on readability. This is especially true during code reviews when you do not have access to "go-to-definitions" and other text editor aids.

Reply:
I didn't actually come up with that name; it was already there before. But sure, I'll change it.